manifests/k/KDE/Kate/21.08.2/KDE.Kate.installer.yaml

```
@@ -1,14 +1,14 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-PackageIdentifier: KDE.Kate
-PackageVersion: 21.08.2
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
+PackageIdentifier: KDE.Kate.Release
+PackageVersion: master-1459
+MinimumOSVersion: 
+InstallerType: 
+Scope: 
 UpgradeBehavior: install
-Installers:
+Installers: 
```

There's some extra whitespace here
```suggestion
Installers:
```